transport-native-unix-common-tests/pom.xml

```diff
@@ -46,4 +46,16 @@
       <scope>compile</scope>
     </dependency>
   </dependencies>
+
+  <build>
```

Comment:
yeah, you are right.  There is only need delete the property 'maven.deploy.skip'. and this was corrected.